File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/CreateTestingRequestIngredientsItem.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class CreateTestingRequestIngredientsItem {
    @JsonProperty("Package")
    public String package_;
    @JsonProperty("Quantity")
    public Integer quantity;
    @JsonProperty("UnitOfMeasure")
    public String unitOfMeasure;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/CreateTestingRequestIngredientsItem.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class CreateTestingRequestIngredientsItem(
    @JsonProperty("Package")
    val package_: String? = null,
    @JsonProperty("Quantity")
    val quantity: Int? = null,
    @JsonProperty("UnitOfMeasure")
    val unitOfMeasure: String? = null
)
